bogota-abril-2019/css-magazine-cover/laura_bruges-task2-magazine/styles/magazine.css

```diff
@@ -0,0 +1,167 @@
+#margin {

```

Comment:
Did you try your code in a different browser? I'm seeing no CSS reset here...

Reply:
Yes, I have tried this out in firefox and chrome. You're right, it was a bit odd since there were some issues with the media specifications for different screen resolutions (I took into account 3 different resolutions). I've just fixed it 😀